src/parser/types.ts

```diff
@@ -516,9 +638,31 @@ interface RandomInterruption {
   components: (string)[];
 }
 
+/**  The InterruptionBlock interface is used to define interruptions in a block. These can be used for breaks or attention checks. Interruptions can be deterministic or random. */
 export type InterruptionBlock = DeterministicInterruption | RandomInterruption;
 
-/** The IndividualComponentSingleResponseCondition interface is used to define a SkipCondition based on a single answer to a specific component. If the component is repeated within the block, this condition will only check the first instance of the component once the order is flattened. */
+/** The IndividualComponentSingleResponseCondition interface is used to define a SkipCondition based on a single answer to a specific component. If the component is repeated within the block, this condition will only check the first instance of the component once the order is flattened.
+ *
+ * For example, if you want to skip to a different component based on the a response to a specific component, you would use the IndividualComponentSingleResponseCondition. Here's an example of how to use the IndividualComponentSingleResponseCondition:
```

Comment:
```suggestion
 * For example, if you want to skip to a different component based on a response to a specific component, you would use the `IndividualComponentSingleResponseCondition`. Here's an example of how to use the `IndividualComponentSingleResponseCondition`:
```

Reply:
Removed the ` characters. This would be the only place that we'd use it. If we want to change it across the whole file, let's do it all together as 1 change